csharp/related.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <PublishAot>true</PublishAot>
```

Comment:
This flag requires explicit RID on publish like
`dotnet publish -r linux-arm64 -c Release`

Also please make sure both build scripts are working correctly and executables paths are correct
https://github.com/jinyus/related_post_gen/blob/main/run.ps1#L81-L93

https://github.com/jinyus/related_post_gen/blob/9db7a779bc431ec5b0d23e25e958bdf3eedd0128/run.sh#L374-L386

Reply:
I didn't modify build scripts as I only tested building from VS on Windows. I'm not familiar with hyperfine, but I'll try dotnet publish command...

Reply:
It builds but output folder of AOT build is changed

Reply:
I added output paths to scripts. I think it should work now.